tempest/lib/javelin

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -x
+#---------------------------------------------------------------
+# Project         : Javelin
+# File            : javelin
+# Copyright       : (C) 2015 by
+# Author          : Joe H. Rahme
+# Created On      : Jan 23 2015
+# Purpose         : Use Javelin for testing
+#---------------------------------------------------------------
+
+source /etc/config-tools/openrc.sh
+
+tempest_dir="/usr/share/openstack-tempest-juno"
+resourcefile="javelin-resource.yml"
+lastjavelin="/opt/javelin-latest-run"   # Make sure that this file survives upgrades by mentioning it in edeploy metadata
```

Comment:
I would rather put the file in /etc/config-tools/javelin or in /opt/javelin/last run (and add /opt/javelin part of eDeploy metadatas to avoid an override during upgrade).